eris.go

```diff
@@ -201,8 +201,11 @@ func (e *wrapError) Format(s fmt.State, verb rune) {
 }
 
 func (e *wrapError) Is(target error) bool {
-	if err, ok := target.(*wrapError); ok {
-		return e.msg == err.msg
+	switch t := target.(type) {
+	case *rootError:
+		return e.msg == t.msg
```

Comment:
should we compare the external error message as well in this case? So a case where root error message doesn't match, we fallback to compare it with external error's msg ?

Reply:
i think that's already covered because `Is` unwraps root errors and continues on to external errors. might be worth adding a test case to verify it though

Reply:
oh i didn't know that Is unwraps root error as well

Reply:
agree that a test case would be beneficial

Reply:
yeah it keeps unwrapping until it hits nil. i'll try to add a test case for this though unless it's already covered

Reply:
this case was already covered